modules/scripts/config.py

```diff
@@ -138,6 +140,7 @@ def _get_env_dict(conf):
             'SWAGGER_UI': extras['swagger']['ui'],
 
             'JAVADOC_PATH': extras['javadoc']['path'],
+            'FEED_URL': social['rss']
```

Comment:
Here also remove this from `.yaydoc.yml`. And document the directive in the README